.circleci/config.yml

```
@@ -228,13 +235,13 @@ workflows:
             - request-staging-approval
 
       - request-acceptance-tests-approval:
-          <<: *main_branch
+          <<: *feature_branch
           type: approval
           requires:
             - deploy_staging
```

```suggestion
            - deploy_staging_preview
```
I think changing it to this will make it work on feature branch deploy 🤞